lib/src/test/java/uk/gov/di/data/lep/library/services/MapperTest.java

```diff
@@ -11,6 +11,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
This seems like a curious addition to me

Reply:
```suggestion
```